### ISSUE TITLE
Missing information in activitie's modal

### ISSUE THREAD

**Issue by ecureuill** [solex2006/SELIProject#36]
When an activity is rendered, in the student side, it shows some important informations. But when the student clicks to open the activity (a modal), this important information is not presented in the modal. It can created difficulties for students with cognitive disabilities.

## Expected Behavior

All the information must be presented in both of the screens. 

## Current Behavior

The information are only presented in the first screen.

![image](https://user-images.githubusercontent.com/993369/78607984-741f5900-7836-11ea-8946-51520fc6acec.png)

![image](https://user-images.githubusercontent.com/993369/78607857-40443380-7836-11ea-977a-a6e0eb2e8668.png)

## Steps to Reproduce

1. Navigate to a course
2. Open a activity

## Environment

Browser: all 

## Possible Solution

Add the information to both the screens as indicated in the image
See #51
